src/Report/Html/Renderer/Template/dashboard.html.dist

```diff
@@ -4,23 +4,21 @@
   <meta charset="UTF-8">
   <title>Dashboard for {{full_path}}</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link href="{{path_to_root}}.css/bootstrap.min.css" rel="stylesheet" type="text/css">
+  <link href="{{path_to_root}}.css/bootstrap.min.css?v=4.1.3" rel="stylesheet" type="text/css">
```

Comment:
The `?v=` everywhere are used for cache bursting in case people serve this from a web server that's caching (too) aggressively.

Reply:
What would you like instead? Getting the new HTML with the old CSS / JS is not going to work.

Reply:
Okay fair enough. I'll remove it.